.changeset/lovely-poems-remember.md

```diff
@@ -0,0 +1,44 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+- Added data attribute 'data-iui-overflow' - when true it adds styling for overflow tabs
+- Added property 'overflowButton' - enables tabs to be truncated and an overflow button to be shown; expects a function that takes the number of items that are visible and returns the 'ReactNode' to render
+
+```typescript
```

Comment:
Did you mean to put this code in the changeset?  I haven't seen us do this before, should we?

Reply:
We do: https://github.com/iTwin/iTwinUI/pull/984/files

I think it's not often necessary when fixing bugs.